From ed1f1a8fa54f6eb7cb297c717ce4c1e85cd343fb Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Thu, 2 Jan 2025 07:53:42 +0000
Subject: [PATCH] Using org.apache.http.legacy.stubs

"org.apache.http.legacy" is deprecated and cannot be used.
So using "org.apache.http.legacy.stubs" instead.

Tests Done: Build and boot

Tracked-On: OAM-128386
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 camera/MultiCameraApplication/Android.bp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/camera/MultiCameraApplication/Android.bp b/camera/MultiCameraApplication/Android.bp
index e5070ed..296dfd2 100644
--- a/camera/MultiCameraApplication/Android.bp
+++ b/camera/MultiCameraApplication/Android.bp
@@ -21,7 +21,7 @@ android_app {
         "com.google.android.material_material",
     ],
 
-    optional_uses_libs: ["org.apache.http.legacy"],
+    optional_uses_libs: ["org.apache.http.legacy.stubs"],
 
     srcs: [
         "java/**/*/*.java",
@@ -45,7 +45,7 @@ android_app {
 
 
     // Guava uses deprecated org.apache.http.legacy classes.
-    libs: ["org.apache.http.legacy"],
+    libs: ["org.apache.http.legacy.stubs"],
 
     jni_libs: [
         "libjni_tinyplanet",
-- 
2.34.1

